src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,14 @@
+.stars {
+  display: flex;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    margin: 0 4px 0 0;
+    background: url(../images/star.svg) no-repeat center;
+  }
+}
+
+.stars__star:nth-child(-n + 4) {
+  background: url(../images/star-active.svg) no-repeat center;
+}
```

Comment:
Use nesting, and it should be the same functionality as in the stars task, if the modifier is `stars--3` you should show 3 active stars and so on